package.json

```diff
@@ -7,19 +7,20 @@
     "@electron/docs-parser": "^0.2.2",
     "@electron/typescript-definitions": "^8.3.1",
     "@octokit/rest": "^16.3.2",
+    "@primer/octicons": "^9.1.1",
```

Comment:
I feel as though this will break a thing / require a change in the default app?

Reply:
i considered this also, but couldn't find one and nothing seemed broken 🤔

Reply:
yep, pushed!